### ISSUE TITLE
Should I install fastclick by myself?

### ISSUE THREAD

**Issue by mrdulin** [Lucifier129/react-lite#130]
I do not see `fastclick` list on `devDependencies` of `package.json` .

Should I install `fastclick` by myself? Or, just add alias 'react-tap-event-plugin': 'react-lite/lib/react-tap-event-plugin' ?

`react-lite/lib/react-tap-event-plugin.js`:

```js
'use strict';

var Fastclick = require('fastclick');
module.exports = function injectTapEventPlugin() {
	var supportTouch = ('ontouchstart' in document);
	if (supportTouch) {
		Fastclick.attach(document.body);
	}
};
```

**Comment by Lucifier129** [comment]
yes, `npm install --save fastclick` is necessary when use `react-lite/lib/react-tab-event-plugin`.

**Comment by mrdulin** [comment]
@Lucifier129 Why don't include `fastclick` as `react-lite` dependency?

**Comment by Lucifier129** [comment]
@mrdulin `react-lite/lib/react-tab-event-plugin` is not the normal usage of `react-lite`, I think it's better to keep independent in `package.json`.
